example/main.cpp

```diff
@@ -46,7 +46,28 @@ int main(int argc, char *argv[])
     initMainWindow(&mainWindow, editor, usePlainTextEdit ? "QPlainTextEdit" : "QTextEdit");
 
     // Connect slots to FakeVimHandler signals.
-    connectSignals(&handler, &mainWindow, editor, fileToEdit);
+    Proxy *proxy = connectSignals(&handler, &mainWindow, editor);
+
+    QObject::connect(proxy, &Proxy::handleInput,
+        handler, [handler] (const QString &text) { handler->handleInput(text); });
```

Comment:
The `handler` is not a pointer and copying QObjects is disabled:

```c++
    QObject::connect(proxy, &Proxy::handleInput,
        &handler, [&handler] (const QString &text) { handler.handleInput(text); });
```

Reply:
... for some reason I didn't enable building examples while testing, so that's embarassing.